File: RAI/EjemploBanco/JavaImplementation/Credito.java
public class Credito extends Producto{
    private double interes;
    private int monto;
    private int cuotas;

    public Credito(int id, double interes, int monto, int cuotas){
        super(id);
        this.interes = interes;
        this.monto = monto;
        this.cuotas = cuotas;
    }

    public String toString(){
        return super.toString() + "Credito: " + this.monto + " (" +this.interes + " en " + this.cuotas + ")";
    }
}

File: RAI/EjemploBanco/python_implementation/Credito.py
from Producto import Producto

class Credito(Producto):
    def __init__(self, identificador, interes, monto, cuotas):
        super().__init__(identificador)
        self.interes = interes
        self.monto = monto
        self.cuotas = cuotas

    def __str__(self):
        return super().__str__() + "Credito: " + str(self.monto) + " (" + str(self.interes) + " en " + str(self.cuotas)
